projects/TR3D/README.md

```diff
@@ -0,0 +1,43 @@
+# TR3D: Towards Real-Time Indoor 3D Object Detection
```

Comment:
The contents in this README.md looks good.
To let users better understand its status and usages in projects, we also recommend developers to follow this [template](https://github.com/open-mmlab/mmdetection3d/blob/dev-1.x/projects/example_project/README.md) to provide more information.